core/command/create.js

```diff
@@ -10,24 +10,35 @@ const template = require('gulp-template')
 
 const { installDependencies } = require('../lib/npm')
 
-async function handler ({ cwd, directory: folderName = 'slides', yes: isDefault }) {
+async function handler ({
+  cwd,
+  directory: folderName = 'slides',
+  yes: isDefault,
+  force: isOverride = false,
+  theme: themeName = '',
+  ratio: aspectRatio = ''
+}) {
```

Comment:
Now the command line parameters do not overlap with what we will ask the user interactively. After adding new parameters, this border is erased.

I would suggest making a general way to override parameters from interactive mode.

By code:
- before the `handle` function describe the object with the parameters (like theme and ratio);
- from them, generate optional parameters for yargs (what is passed to options);
- generate params from them for interactive mode.

If the user has defined a variable in the command-line arguments, then do not ask again interactively.

Check that the parameters are valid. If not valid, then fall with an error.

Optionally, you could add the `--no-interactive` parameter to not enter interactive mode, but to crash with an error if not all parameters are passed to the CLI.

--

What do you think about it? Do you need help? You can write to me in PM (https://t.me/nikolay_govorov) if you have any questions.